sklearn/cluster/_dbscan.py

```diff
@@ -270,8 +270,8 @@ class DBSCAN(ClusterMixin, BaseEstimator):
     DBSCAN revisited, revisited: why and how you should (still) use DBSCAN.
     ACM Transactions on Database Systems (TODS), 42(3), 19.
     """
-
-    def __init__(self, eps=0.5, min_samples=5, metric='euclidean',
+    @_deprecate_positional_args
+    def __init__(self, eps=0.5, *, min_samples=5, metric='euclidean',
```

Comment:
I don't know much about dbscan so I don't have any opinion, but

is `DBSCAN(0.6)` something that would be non-ambiguous for someone experienced with DBSCAN?
I.e. is it obvious that this is `DBSCAN(eps=0.6, metric='euclidean')`?

Reply:
It's not completely obvious, but this is certainly the key parameter in dbscan.

Reply:
Also the more realistic usecase is not so much `DBSCAN(0.6)`, but
```py
def cluster(eps=0.6):
   ...
   ... = DBSCAN(eps)
```